test/e2e/functional_test.go

```diff
@@ -93,6 +97,69 @@ func (s *FunctionalSuite) TestFastFailOnPodTermination() {
 		})
 }
 
+func (s *FunctionalSuite) TestEventOnNodeFail() {
+	s.Given().
+		Workflow("@expectedfailures/failed-step-event.yaml").
+		When().
+		SubmitWorkflow().
+		WaitForWorkflow(120 * time.Second).
```

Comment:
does this need 2m? it's long and I worry we'll end up with an e2e suite that takes too long to run - can you see if 30s works?

Reply:
tested, seems to work fine. updated :)